cloud/grouping.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+import yaml
+
+import cloud
+
+from cloudferrylib.os.identity import keystone
+from cloudferrylib.os.network import neutron
+from cloudferrylib.os.compute import nova_compute
+
+
+class Grouping(object):
```

Comment:
Only `group()` method seem to be public. Let's stick with python style of marking all private/protected methods with underscore.